storage/tests/system.py

```diff
@@ -1704,6 +1705,8 @@ def test_bpo_set_unset_preserves_acls(self):
         bucket.iam_configuration.bucket_policy_only_enabled = True
         bucket.patch()
 
+        self.assertTrue(bucket.iam_configuration.bucket_policy_only_enabled)
```

Comment:
Is this expected?

Reply:
Yes:  the bucket's properties get updated via the resource returned from the PATCH request.

Reply:
Sg, thanks. Please move forward when ready.